packages/svelte/tests/migrate/samples/state-and-derivations-sequence/output.svelte

```diff
@@ -0,0 +1,14 @@
+<script>
+	let count = ($state((void 0, 0)));
```

Comment:
this output doesn't seem right

```suggestion
	let count = $state((void 0, 0));
```

Reply:
The migration simply append `$state(` and `)` to the left and right of the `init` (it was already like this before).

This keeps the code very simple but if you have multiple parentheses spit out some weird code. Eg.

```ts
let count = (((0)));
```
converts to
```ts
let count = ((($state(0))));
```
Because parentheses are basically counted nowhere in the ast.

We can probably fix this with some more complex code to migrate state but I'm not sure is worth it just for the esthetic.

Reply:
So basically the point is: the last tells us where the expression is but the expression does not include the parentheses and we can't really be sure of how many parentheses, whitespace, code that the ast doesn't consider are there so the code will be probably much more complex (and even worse for handling the derived)

Reply:
i fixed it 😉

Reply:
Ugh I need to remember that doing such things is fine...whenever I'm working with some AST i tend to avoid touching strings as much as possible but it's actually totally fine. Thanks and sorry 😉